src/hardware/mixer.cpp

```diff
@@ -2472,61 +2508,88 @@ static void mixer_thread_loop()
 		assert(mixer.state != MixerState::Uninitialized);
 
 		// This code is mostly for the fast-forward button (hold Alt + F12)
-		const double now = PIC_FullIndex();
+		const double now         = PIC_FullIndex();
 		const double actual_time = now - last_mixed;
-		const double expected_time = (static_cast<double>(mixer.blocksize) / static_cast<double>(mixer.sample_rate_hz)) * 1000.0;
+		const double expected_time = (static_cast<double>(mixer.blocksize) /
+		                              static_cast<double>(mixer.sample_rate_hz)) *
+		                             1000.0;
 		last_mixed = now;
 
 		// "Underflow" is not a concern since moving to a threaded mixer
-		// If the CPU is running slower than real-time, the audio drivers will naturally slow down the audio
-		// Therefore, we can always request at least a blocksize worth of audio
+		// If the CPU is running slower than real-time, the audio
```

Comment:
Again, full stops or manual formatting is needed here.

Reply:
Cheers, I fixed up a few but did not read all comments. @weirddan455 I'd suggest to get into the habit of using punctuation in comments 😄 These were all yours from the mixer refactoring. Very helpful comments, so please keep commenting; just a minor suggestion to help with reformatting 😄